.github/workflows/build-and-deploy.yml

```diff
@@ -38,6 +38,26 @@ jobs:
           name: build-artifacts
           path: dist
 
+  formatting:
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18
+
+      - name: Run install
+        uses: borales/actions-yarn@v5
+        with:
+          cmd: install
+
+      - name: Check Formatting
```

Comment:
Wondering if this can be part of the previous job instead?

Reply:
We don't want it to block build step, it can run in parallel